pkg/services/ngalert/models/alert_rule.go

```diff
@@ -757,3 +766,26 @@ func GroupByAlertRuleGroupKey(rules []*AlertRule) map[AlertRuleGroupKey]RulesGro
 	}
 	return result
 }
+
+// Record contains mapping information for Recording Rules.
+type Record struct {
```

Comment:
The field name or the struct name? It's a field called `Record`, of type `Record` right now

Reply:
I think field name should remain (`record` also has meaning in prometheus). Struct name i'm totally good with changing if we want - but this can be done without DB migration anytime in future